content/events/amia-informatics-summit-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AMIA"
+description: "Attend AMIA 2023 Informatics Summit to experience content dedicated to translational bioinformatics, precision medicine, clinical research informatics, data science and artificial intelligence (AI) in one integrated conference."
+eventType: "Conference"
+featured: true
+hashtag: "#IS23"
+location: "Seattle, WA, USA"
+sessions: [{sessionStart: "13 March 2023"}, {sessionStart: "14 March 2023"}, {sessionStart: "15 March 2023"}]
+timezone: "America/New_York"
```

Comment:
Let's confirm with @NoopDog the correct `tz` for Seattle (see momentjs)[https://momentjs.com/timezone/]